releases/release-1.31/release-notes/maps/pr-124063-map.yaml

```diff
@@ -0,0 +1,7 @@
+pr: 124063
+releasenote:
+  text: Fixed a bug where hard evictions due to resource pressure would let the pod
+    have the full termination grace period, instead of shutting down instantly. This
```

Comment:
```suggestion
    have the full termination grace period, instead of shutting down instantly. This
```
```suggestion
    to use the full termination grace period instead of shutting down instantly. 
```